### PR TITLE
Fixed Care Pathways custom reports

### PR DESCRIPTION
**Pull request by kkrampa**
@czue http://manage.dimagi.com/default.asp?211968#1080131

### CODE REVIEW

File: custom/care_pathways/sqldata.py [hunk 1]
```diff
@@ -72,11 +74,12 @@ def _build_query(self, table, filter_values):
         table_card_group = []
         if 'group_name' in self.group_by:
             table_card_group.append('group_name')
-        s1 = alias(
-            select(
-                ['doc_id', 'group_id', 'MAX(prop_value) + MIN(prop_value) as maxmin'] + filter_cols +
-                external_cols, from_obj='"fluff_FarmerRecordFluff"',
-                group_by=['doc_id', 'group_id'] + filter_cols + external_cols
+        s1 = alias(select(
+                [table.c.doc_id, table.c.group_id,
```
Comment (dimagimon) [inline custom/care_pathways/sqldata.py]: E126 continuation line over-indented for hanging indent

**Comment by kkrampa** [comment]
@czue Could you kick build again?

**Comment by czue** [comment]
kicked and it failed again. just tried one more time. it might need to be merged with master if that doesn't work

**Comment by czue** [comment]
was this an upgrade issue?

**Comment by kkrampa** [comment]
Yep

**Pull request merged**